app/lib/controlled_work_mappings/cw1_new.yml

```diff
@@ -0,0 +1,441 @@
+# This file defines the mappings used to prepopulate a CW1 PDF form with the answers given by a user to CCQ's questions
+# combined with the output from CFE.
+
+# The mappings conform to the format described in ControlledWorkDocumentValueMappingService
+- section: general
+  fields:
+    # "Means test required"
+    - name: Means test required
+      type: always_checked_checkbox
+      checked_value: "Yes_2"
+
+    # "The client is a child under the age of 18:"
+    - name: Child under 18
+      type: boolean_radio
+      source: under_eighteen?
+      yes_value: "Yes_2"
+      no_value: "No"
+
+    # "The child’s means are to be aggregated with a maintaining adult."
+    - name: Means to be aggregated with maintainng adult
```

Comment:
'maintaining' instead of 'maintainng'

Reply:
still showing wrong spelling?

Reply:
this is a spelling error in the PDF and as it is the name of the field we have to reproduce it exactly the same way here.